{
    "config": {
        "abort": {
            "already_configured": "Dienst ist bereits konfiguriert",
            "cannot_connect": "Verbindung fehlgeschlagen"
        },
        "error": {
            "cannot_connect": "Verbindung fehlgeschlagen",
            "unknown": "Unerwarteter Fehler"
        },
        "step": {
            "user": {
                "data": {
                    "station_id": "Hydrologische Station"
                },
                "data_description": {
                    "station_id": "W\u00e4hle eine hydrologische Station aus der Liste aus."
                }
            }
        }
    },
    "entity": {
        "sensor": {
            "water_level": {
                "name": "Wasserstand"
            },
            "water_temperature": {
                "name": "Wassertemperatur"
            }
        }
    },
    "exceptions": {
        "cannot_connect": {
            "message": "Beim Verbinden mit der IMGW-PIB-API f\u00fcr {entry} ist ein Fehler aufgetreten: {error}"
        },
        "update_error": {
            "message": "Beim Abrufen der Daten von der IMGW-PIB-API f\u00fcr {entry} ist ein Fehler aufgetreten: {error}"
        }
    }
}